src/ns.c

```diff
@@ -132,10 +132,16 @@ SDB_API bool sdb_ns_unset (Sdb *s, const char *name, Sdb *r) {
 	if (s && (name || r)) {
 		ls_foreach (s->ns, it, ns) {
 			if (name && (!strcmp (name, ns->name))) {
+				sdb_free (ns->sdb);
+				free (ns->name);
+				free (ns);
```

Comment:
Shouldn't this be just `sdb_ns_free()`? Though that function looks weird.

Reply:
> Shouldn't this be just `sdb_ns_free()`?

Yes 15f77c95f191bb1091901c6d0c2c7a24595e9412.

> Though that function looks weird.

I've renamed some functions in 73815f564599b7ddb3682b8e2aad6aeb340d5987 based on what I think their intended purpose is.